dictionaries/CallMap.php

```diff
@@ -8462,16 +8462,20 @@
 'mysqli_error_list' => ['array', 'mysql'=>'mysqli'],
 'mysqli_escape_string' => ['string', 'mysql'=>'mysqli', 'string'=>'string'],
 'mysqli_execute' => ['bool', 'statement'=>'mysqli_stmt', 'params='=>'list<mixed>|null'],
-'mysqli_fetch_all' => ['array', 'result'=>'mysqli_result', 'mode='=>'int'],
-'mysqli_fetch_array' => ['array|false|null', 'result'=>'mysqli_result', 'mode='=>'int'],
-'mysqli_fetch_assoc' => ['array<string,string>|false|null', 'result'=>'mysqli_result'],
+'mysqli_fetch_all' => ['list<array<array-key,null|int|float|string>>', 'result'=>'mysqli_result', 'mode='=>'MYSQLI_BOTH'],
+'mysqli_fetch_all\'1' => ['list<array<string,null|int|float|string>>', 'result'=>'mysqli_result', 'mode='=>'MYSQLI_ASSOC'],
+'mysqli_fetch_all\'2' => ['list<list<null|int|float|string>>', 'result'=>'mysqli_result', 'mode='=>'MYSQLI_NUM'],
```

Comment:
Psalm does not support non-class constants as argument types (they can't be syntactically distinguished from class names), so you'd need to use their values instead.